src/test/java/org/minions/devfund/mauricioz/linkedList/ContainsTest.java

```diff
@@ -0,0 +1,341 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ Source code recreated from a .class file by IntelliJ IDEA
+ (powered by Fernflower decompiler)
+ */
+public class ContainsTest {
+    public ContainsTest() {
+    }
+
+    /**
+     * setup for tests.
+     * @throws Exception
+     */
+    @Before
+    public void setUp() throws Exception {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)